modules/components/pages/inputs/http_client.adoc

```diff
@@ -760,6 +760,18 @@ A HTTP proxy URL (optional).
 
 *Type*: `string`
 
+=== `disable_http2`
+
+Whether to disable disable HTTP/2. By default, HTTP/2 is enabled.
+
+*Type*: `bool`
+
+*Default*: `false`
+
+ifndef::env-cloud[]
+Requires version 4.44.0 or later
```

Comment:
Nit: That should be [v4.47.0](https://github.com/redpanda-data/connect/releases/tag/v4.47.0). `4.44.0` is the Benthos framework version, but people will use this through Connect.